public/employers/honda.yml

```diff
@@ -0,0 +1,34 @@
+name: Honda Motor Company, Ltd.
+shortName: Honda
+ticker: HMC
+wiki: Honda
+image: honda.svg
+officialWebsite: https://www.honda.com/about
+location:
+  city: Minato City
+  country: jp
+  international: true
+  wiki: Minato, Tokyo
+employeesBefore:
+  type: exactly
+  upperBound: 215638
+  year: 2018
+summary: Honda has responded to the crisis by halting production in North America and deep-cleaning its facilities, without detrement to its workers' salaries or benefits.
```

Comment:
It might be useful to say what they're doing elsewhere in the world, even if we don't know